src/ui/components/SearchInput/style.scss

```diff
@@ -56,7 +56,7 @@ $transition-duration: 250ms;
   // Include the start margin on the end so that icon and text is centred together.
   @include margin-end($icon-margin-start);
 
-  color: #ccc;
+  color: #fff;
```

Comment:
Should we use `$base-color` here instead? `$base-color` is `#fff` and presumably we'd want to update most #fff's across the board as our "base white" if/when we changed this.